### PR TITLE
Format files

### PR DESCRIPTION
**Pull request by 6543**
as title ...

It's anoing to see many lines of diff if you edit one of this files and the editor automaticaly correct unrelated things.

this pull format files so there will be less + fix alignment of "make help"

**this pull does not change any behaviour**

### CODE REVIEW

File: docs/.editorconfig [hunk 1]
```diff
@@ -11,7 +11,7 @@ trim_trailing_whitespace = true
 indent_style = tab
 indent_size = 8
 
-[*.{tmpl,html}]
+[*.{tmpl, html}]
 indent_style = tab
 indent_size = 4
```
Comment (silverwind) [inline docs/.editorconfig]: May want to merge this file with the one in root (and update any files in `docs` to conform).

File: .eslintrc [hunk 2]
```diff
@@ -28,375 +28,375 @@ globals:
...
+  vars-on-top: [ 0 ]
+  vue/attributes-order: [ 0 ]
+  vue/component-definition-name-casing: [ 0 ]
+  vue/html-closing-bracket-spacing: [ 0 ]
+  vue/max-attributes-per-line: [ 0 ]
+  vue/one-component-per-file: [ 0 ]
+  wrap-iife: [ 2, inside ]
+  wrap-regex: [ 0 ]
+  yield-star-spacing: [ 2, after ]
+  yoda: [ 2, never ]
```
Comment (silverwind) [inline .eslintrc]: I don't particularily like this style of spacing and we don't do it in JS either. Maybe rename to `.eslintrc.yaml` to see if that shuts up your tool?

File: build/generate-svg.js [hunk 3]
```diff
@@ -50,7 +50,8 @@ async function processFile(file, {prefix = ''} = {}) {
 async function main() {
   try {
     await mkdir(outputDir);
-  } catch {}
+  } catch {
+  }
```
Comment (silverwind) [inline build/generate-svg.js]: Is it really an issue? eslint certainly doesn't complain about it.

Comment (6543) [reply build/generate-svg.js]: it's not - the importand thing for me was about markdown and ini files :)

File: .eslintrc (hunk 2 above)
Comment (6543) [reply .eslintrc]: @silverwind is `.eslintrc.yaml` supported by our js linter?

Comment (6543) [reply .eslintrc]: never mind ... it does not work anyway ... - reverting

File: docs/.editorconfig (hunk 1 above)
Comment (6543) [reply docs/.editorconfig]: @silverwind done

File: .editorconfig [hunk 4]
```diff
@@ -20,3 +22,9 @@ insert_final_newline = false
 
 [*.md]
 trim_trailing_whitespace = false
+
+[*.go]
+indent_size = 8
```
Comment (lafriks) [inline .editorconfig]: Ident size 8 imho is way too much 😰

File: .editorconfig [hunk 5]
```diff
@@ -20,3 +22,9 @@ insert_final_newline = false
...
+
+[*.{tmpl, html, less, js}]
+indent_size = 4
```
Comment (lafriks) [inline .editorconfig]: For javascript most defaults to 2 spaces identing

File: .eslintrc (hunk 2 above)
Comment (silverwind) [reply .eslintrc]: `.eslintrc.yaml` should generally work except for maybe some outdated editor integrations which is the reason I generally use the older `.eslintrc`.

File: .editorconfig (hunk 4 above)
Comment (silverwind) [reply .editorconfig]: That section can just be removed, we use size 2 for go since a while now inherited from `[*]`.

File: .editorconfig (hunk 5 above)
Comment (silverwind) [reply .editorconfig]: I'd remove this as well and reindent matching files in `docs`. The other formats should already be at 2-space outside that folder.

File: .editorconfig [hunk 6]
```diff
@@ -1,3 +1,5 @@
+# http://editorconfig.org
+
```
Comment (silverwind) [inline .editorconfig]: Remove as well, I remember specifically removing this advertisment 😉

**Review by silverwind — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Review by zeripath — APPROVED** [review]

**Comment by lafriks** [comment]
🚀

**Comment by 6543** [comment]
:rocket:

**Pull request merged**